_posts/2025-02-27-defna-board-member-recruitment.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title:  "Join DEFNA! Board Member Recruitment"
+date: 2025-02-27 12:30:00
+categories: ""
+author_name : Velda Kiara
+author_url : /about
+author_avatar: defna
+show_avatar : true
+read_time : ""
+feature_image: 2025-02-27
+show_related_posts: false
+permalink: /announcements/2025/02/27/defna-board-member-recruitment/
+---
+
+**Join the DEFNA Board as Corporate Secretary!**
```

Comment:
Do we want to require that they be the secretary?

Reply:
Agreed with Velda. The incoming director should be aware of the expectations and responsibilities of not only a director but also a secretary. We need organization and verbosity from a detail oriented person.